frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -90,7 +94,10 @@ export class HtmlDocument extends React.Component {
       invalidMentionList: [],
       oldInvalidMentionList: [],
       showInvalidMentionPopupInComment: false,
-      showInvalidMentionPopupInContent: false
+      showInvalidMentionPopupInContent: false,
+      translatedRawContent: null,
+      defaultTranslationState: defaultTranslationState,
```

Comment:
It seems this is never changed, maybe this should be kept away from the state?

Reply:
I hesitated, theoretically it could change as the default value comes from a property.
And it is unsure where to keep this if it is not in the state: here I could put it in the component's object but wouldn't work with a pure component (and later on with hooks).